<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QWidget" name="SettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="general">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <widget class="QCheckBox" name="itcCache">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Enable login session restoring</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="condensedViews">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>As an example, &quot;135,230 views&quot; would be condensed to &quot;135.2k views.&quot;</string>
       </property>
       <property name="text">
        <string>Use condensed view counts</string>
       </property>
      </widget>
      <widget class="QLabel" name="volumeLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>220</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Preferred player volume</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="volumeSpin">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>208</y>
         <width>70</width>
         <height>34</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
      <widget class="QCheckBox" name="homeShelves">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>As an example, &quot;135,230 views&quot; would be condensed to &quot;135.2k views.&quot;</string>
       </property>
       <property name="text">
        <string>Use shelves homepage layout</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="returnDislike">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>As an example, &quot;135,230 views&quot; would be condensed to &quot;135.2k views.&quot;</string>
       </property>
       <property name="text">
        <string>Enable 'Return Youtube Dislike' integration</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="fullSubs">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>As an example, &quot;135,230 views&quot; would be condensed to &quot;135.2k views.&quot;</string>
       </property>
       <property name="text">
        <string>Use full subscriber counts</string>
       </property>
      </widget>
      <widget class="QLabel" name="defaultTabLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>220</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Default front page tab</string>
       </property>
      </widget>
      <widget class="QComboBox" name="defaultTabCombo">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>248</y>
         <width>135</width>
         <height>34</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Home</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Trending</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Subscriptions</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>History</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="privacy">
      <attribute name="title">
       <string>Privacy</string>
      </attribute>
      <widget class="QCheckBox" name="playbackTracking">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Playback tracking allows the user to add to a video's view count when the video is played.</string>
       </property>
       <property name="text">
        <string>Enable playback tracking</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="watchtimeTracking">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Watch time tracking allows the History tab to function and is used in creator analytics.</string>
       </property>
       <property name="text">
        <string>Enable watch time tracking</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="sponsorblock">
      <attribute name="title">
       <string>SponsorBlock</string>
      </attribute>
      <widget class="QLabel" name="blockLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Block:</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="blockSponsor">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>250</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Sponsors</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="blockIntro">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>250</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Intros</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="blockOutro">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>250</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Endcards/Credits</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="blockSelfPromo">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>250</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Self Promotion</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="blockPreview">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>40</y>
         <width>224</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Previews/Recaps</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="blockFiller">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>70</y>
         <width>224</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Filler Tangents/Jokes</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="blockInteraction">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>100</y>
         <width>224</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Interaction Reminders</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="blockNonMusic">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>130</y>
         <width>224</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Non-Music Sections</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="showToasts">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>400</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Show toasts (popups, notifications)</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
